### ISSUE TITLE
Won't work with filename with spaces in the name

### ISSUE THREAD

**Issue by jjarava** [fabianmoronzirfas/vscode-open-in-marked-two#2]
The extension currently won't work if the file to be opened in Marked has spaces in the name.

Having a look at the extension, the change to fix the issue seems quite minor.

line 27 in `extension.js` should read

```js
let cmd = `open -a "${values.apppath}" "${currentlyOpenTabfilePath}"`;
```

**Comment by jjarava** [comment]
I've created a PR with what I think should be the fix; but please review if the "command to be executed" is defined elsewhere as I'm not familiar with VS Code plugins, etc.

**Comment by fabianmoronzirfas** [comment]
closed by 07102542ca5971da6bff0c2f3530d77f5faaafa1
